src/orca-log-catcher/OrcaLogger.res

```diff
@@ -39,7 +39,7 @@ type eventName =
   | APP_REINITIATED
   | LOG_INITIATED
   | LOADER_CALLED
-  | ORCA_ELEMENTS_CALLED
+  | WEB_SDK_IFRAME_MOUNTED
```

Comment:
Could you please explain the use of this log? Can it be something like `HYPER_ELEMENTS_CALLED` or `ELEMENTS_CALLED`?

Reply:
We wanted to make it more readable to merchant in control center's audit trail. The names like HYPER_ELEMENTS_CALLED or ELEMENTS_CALLED makes more sense to Hyperswitch devs, but we wanted to make it more generic.

Reply:
Could you please rename this to `PAYMENT_ELEMENTS_INITIATED`. Since we should not have WEB as a prefix for any log event. Also since this is not the only iframe which gets mounted, it will be confusing moving forward

Reply:
The name of mobile counter part of this event name is APP_RENDERED so we can use WEB as prefix for this one. Agree, PAYMENT_ELEMENTS_INITIATED is more a generic and umbrella term as it's not only about an iframe. But to a merchant iframe mounted is a more tangible event instead of something like elements initiated. The merchant should be able to derive some sense out of an event name. Let's decide a self explanatory event name for this before moving forward.  cc @seekshiva @akash-c-k @PritishBudhiraja